gammapy/cube/fit.py

```diff
@@ -33,7 +33,12 @@
 MIGRA_AXIS_DEFAULT = MapAxis.from_bounds(
     0.2, 5, nbin=48, node_type="edges", name="migra"
 )
-BINSZ_IRF = 0.2
+ENERGY_AXIS_DEFAULT = MapAxis.from_edges(
```

Comment:
Yes, I did indeed copy it over from `gammapy.spectrum`, and this is clearly a poor choice of binning (which I think we need to take from the IRFs at some point anyways). But I think it might be needed to have a default `e_true` binning. I remember I was getting very different indices between spectral and map based analysis once, which was solved by changing the `e_true`. This would be quite non-intuitive for a general user...

@registerrier @bkhelifi  do you have an opinion about this?

Reply:
Yes, the assumption of the 3D model evaluation is that we use a "fine" binning (I think we never clarified, what this actually means, probably >5 bins / decade, but we should document it somewhere). If this is not fulfilled, the user can currently get bad or unexpected results. I think in general we should change to using `SpectralModel.integral()` in the 3D analysis as well (just as we do for 1D), this should already improve (if not solve...) the discrepancy. Note, that the for the `SpectrumDatasetMaker` I already removed the default for `e_true`. But we can certainly discuss, whether we re-introduce it in a consistent way and with a plausible choice for 1D as well as 3D.

Reply:
I think that the issue is not only the fine binning but the total range.
 In theory, the true energy binning is supposed to cover the 0 to + infinity range. It is of course not necessary in practice, but the range has to be larger than the `e_reco` one to account for spill-over. 
This is far from a small approximation in many cases if users define a range close to the actual safe range.
A possibility could be to decide that by default `e_true` is binned starting from `e_reco[0]*0.7` up to `e_reco[-1]*1.3`. Or even to leave as a possible parameter the leakage fraction (with default at 30% for instance).